ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-aro.yaml

```diff
@@ -24,20 +24,10 @@ base_images:
     namespace: ocp
     tag: tools
   upi-installer:
-    name: "4.11"
+    name: "4.13"
```

Comment:
If we change aro-provision-cluster-ref.yaml from:
```
  from: upi-installer
```
to
```
  from_image:
   namespace: ocp
   name: "4.14"
   tag: upi-installer
```
I think we even do not need to set `upi-installer` base image in the yaml file.